integr8ly/integreatly-operator
# MGDAPI-2576 - Add support for 3scale 2.11

# Issue link
https://issues.redhat.com/browse/MGDAPI-2576

# What
Add support for 3scale 2.11 in RHOAM

# Verification steps
- Check that new 3scale is installed and running  no errors
- Check that 3scale is using all images from quay.io/integreatly/delorean  repo, as defined in manifest 3scale-operator.v0.8.0.clusterserviceversion.yaml
- Check that manifest is using correct images --  integreatly-operator/manifests/integreatly-3scale/0.8.0/3scale-operator.v0.8.0.clusterserviceversion.yaml
- Check as noted in Jira : "Verify that our "authentication delay" plugin works - open 3scale admin route in a new browser session, login with a user from testing-idp that has not been used on the cluster yet, you should see a page saying something like "Your account is being provisioned", after a while (up to 5mins) you should be successfully loged in and redirected to 3Scale"
